src/debug/di/module.cpp

```diff
@@ -880,6 +881,11 @@ HRESULT CordbModule::InitPublicMetaDataFromFile(const WCHAR * pszFullPathName,
                                                 DWORD dwOpenFlags,
                                                 bool validateFileInfo)
 {
+#ifdef FEATURE_PAL    
+    // TODO: Some intricate details of file mapping don't work on Linux as on Windows.
+    // We have to revisit this and try to fix it for POSIX system. 
+    return E_FAIL;
+#else    
```

Comment:
Clarification: I'm not requesting any change to the code above, just adding context